Aspire BudgetingTests/MockUser.swift

```diff
@@ -2,33 +2,30 @@
 //  MockUser.swift
 //  Aspire BudgetingTests
 //
-//  Created by TeraMo Labs on 11/1/19.
-//  Copyright © 2019 TeraMo Labs. All rights reserved.
-//
 
 @testable import Aspire_Budgeting
 import Foundation
+import GoogleSignIn
 import GTMSessionFetcher
 
-final class MockUser: AspireUser {
-  typealias Profile = MockProfile
-
-  typealias Authentication = MockAuthentication
-
-  var profile: Profile! = MockProfile()
-  var authentication: Authentication! = MockAuthentication()
+final class MockProfile: GIDProfileData {
+  override var name: String! {
+    return "Ghanta Singh"
+  }
 }
 
-final class MockProfile: AspireProfile {
-  var name: String! = "First Last"
+class MockAuthentication: GIDAuthentication {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/564f6caae93c3b6163a455d44e2d2feb416590ee/Aspire BudgetingTests/MockUser.swift#L17"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>